graphql/src/main/java/com/scalar/db/graphql/server/GraphQlServer.java

```diff
@@ -0,0 +1,67 @@
+package com.scalar.db.graphql.server;
+
+import com.scalar.db.api.DistributedStorageAdmin;
+import com.scalar.db.config.DatabaseConfig;
+import com.scalar.db.graphql.server.config.ServerConfig;
+import com.scalar.db.service.StorageFactory;
+import java.io.FileInputStream;
+import java.util.Properties;
+import java.util.concurrent.Callable;
+import org.eclipse.jetty.server.Server;
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+
+@Command(name = "scalardb-graphql", description = "Starts Scalar DB GraphQL server.")
+public class GraphQlServer implements Callable<Integer> {
+  @CommandLine.Option(
+      names = {"--properties", "--config"},
+      required = true,
+      paramLabel = "PROPERTIES_FILE",
+      description = "A configuration file in properties format.")
+  private String configFile;
+
+  private ServerConfig config;
+
+  public GraphQlServer() {}
+
+  public GraphQlServer(ServerConfig config) {
+    this.config = config;
+  }
+
+  public static void main(String[] args) {
+    int exitCode = new CommandLine(new GraphQlServer()).execute(args);
+    System.exit(exitCode);
+  }
+
+  @Override
+  public Integer call() throws Exception {
+    if (configFile != null) {
+      Properties properties = new Properties();
+      try (FileInputStream fis = new FileInputStream(configFile)) {
+        properties.load(fis);
+      }
+      config = new ServerConfig(properties);
+    }
```

Comment:
So we can keep this null check and use the constructor `ServerConfig(File propertiesFile)`:
```suggestion
    if (configFile != null) {
      config = new ServerConfig(new File(configFile));
    }
```